base/math.jl

```diff
@@ -715,6 +729,9 @@ Stacktrace:
 julia> sqrt(big(complex(-81)))
 0.0 + 9.0im
 
+julia> sqrt(-81 -0.0im)  # -0.0im is below the branch cut
```

Comment:
```suggestion
julia> sqrt(-81 - 0.0im)  # -0.0im is below the branch cut
```

Reply:
I would normally write a space, but somehow here we're not just subtracting zero, `-0.0` is a special trade-marked *thing*. which seems weird to break up.

Reply:
Although I see I did so elsewhere!